core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -108,7 +108,7 @@ case class ArrayDescriptor(fqn: DescriptorType) extends DescriptorType {
   }
   def internalString = "[" + fqn.internalString
 }
-case class Descriptor(params: List[DescriptorType], ret: DescriptorType) {
+case class Descriptor(params: Seq[DescriptorType], ret: DescriptorType) {
```

Comment:
can you make these `List` please? Or are you relying on this for some performance reason? Note that you end up calling `.toList` on it anyway.